cmd/argocd/commands/cluster.go

```diff
@@ -3,6 +3,7 @@ package commands
 import (
 	"context"
 	"fmt"
+	"github.com/mattn/go-isatty"
```

Comment:
The linter is complaining about the location of this import. The imports are divided in three distinct blocks, each of which must be alphabetically sorted:

1. Golang standard libary
2. Own package imports
3. External package imports

Can you please move the import line to its own block to quiet the linter?